src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -240,7 +246,7 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchFeedItems();
     };
 
-    updateTaskAssignment = (taskId: string, taskAssignmentId: string, status: TaskAssignmentStatus): void => {
+    updateTaskAssignment = (taskId: string, taskAssignmentId: string, status: TaskCollabStatus): void => {
```

Comment:
This was (I believe) incorrect before. Should be using the new uppercase values from `TaskCollabStatus` type. This is the main change in this PR besides explicit imports.